mlflow/store/artifact/cloud_artifact_repo.py

```diff
@@ -29,6 +29,15 @@
 _ARTIFACT_UPLOAD_BATCH_SIZE = (
     50  # Max number of artifacts for which to fetch write credentials at once.
 )
+_AWS_MIN_CHUNK_SIZE = 5 * 1024**2  # 5 MB is the minimum chunk size for S3 multipart uploads
+
+
+def _validate_chunk_size_aws() -> None:
+    chunk_size = MLFLOW_MULTIPART_UPLOAD_CHUNK_SIZE.get()
+    if chunk_size <= _AWS_MIN_CHUNK_SIZE:
```

Comment:
```suggestion
    if chunk_size < _AWS_MIN_CHUNK_SIZE:
```

to allow `5 * 1024**2`.

Reply:
Changed